vivisect/tests/samplecode.py

```diff
@@ -1,23 +0,0 @@
-'''
```

Comment:
did this find a home somewhere?  or was it straight up cut?

Reply:
This particular set of code I cut, mostly because where it was used (testcodegraph.py) wasn't *actually* testing the creation/usability of the Codegraph class.

So instead I pulled in firefox.exe for firefox version, RE'd some functions to manually build out their codegraphs, and added those as tests in testcodegraph instead.